pieforproviders/pieforproviders
# Feature/514/associate child with subsidy rule

# 💅🏼 What issue does this fix?
#514 - when a child is created or updated, they are associated with a subsidy rule based on their age and the county in which they receive care.

## 🍂 Before You Submit


## 🛷 Deployment


## 🧵 Steps to set up locally

```
bundle exec rails db:migrate
```

## 🧳 Steps to test

Using `bundle exec rails console`:
- create a non-expired SubsidyRule
- create a Business in the county that SubsidyRule covers
- create a Child that attends that Business and is the age covered by that SubsidyRule
- `child.current_subsidy_rule` should be the Rule you created

## 🕯 Caveats, concerns
